src/Login.js

```diff
@@ -108,19 +110,28 @@ export default function SignIn() {
       isManager: false,
       isMentor: true
     });
-    // axios.get(`${process.env.REACT_APP_ENV}/user/${uid}`).then(function(response) {
-    //   const { managerId, mentorId } = response.data;
-    //   setuserTypes({
-    //     isManager: managerId ? true : false,
-    //     isMentor: mentorId ? true : false
-    //   });
-    // });
+
+    axios.get(`${process.env.REACT_APP_ENV}/user`, {
+      headers: {
+        Authorization: "Bearer " + idToken
+      }
```

Comment:
Same here, can just call the currentuser.getIdToken instead of the listener above and saving in local state.